src/riscv-integration.adoc

```diff
@@ -763,7 +763,8 @@ xref:mtval-cheri-causes[xrefstyle=short] respectively.
 | 1                | Seal violation
 | 2                | Permission violation
 | 3                | Length violation
-| 4-15             | Reserved
+| 4                | Invalid address violation
```

Comment:
This is what I was trying to say in https://github.com/riscv/riscv-cheri/pull/188#issuecomment-2051486224: up till now the encodings have mirrored the priorities, so should Length be 4 and Invalid address be 3? It seems likely future extensions would be unable to maintain this property though, so maybe it's not worth bothering with. Still, it could be worth having a table of priorities?

Reply:
Fixed